formats/developers/using-ome-xml.txt

```diff
@@ -394,6 +397,9 @@ Possible included objects
     - ``<xs:element ref="Line">``
     - ``<xs:element ref="ListAnnotation">``
     - ``<xs:element ref="LongAnnotation">``
+    - ``<xs:element ref="M">``
```

Comment:
Yep, it is part of the map pairs structure, do you think I need a comment on that line?

It is used like so:
```
<M K="theKey">theValue</>
```